demo/Nonlinear Noisy Pendulum.ipynb

```diff
@@ -0,0 +1,1404 @@
+{
```

Comment:
<p>I don't get this plot. On GitHub I can only see a single distribution. Furthermore, what do we mean with <code>p(\tau \mid y_3)</code>? The posterior with respect to only variable <code>y_3</code> or all variables up till <code>y_3</code> ?  From the code, I understand it as the variational posterior (perhaps better call it <code>q</code> , same below) as a function of the number of iterations.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/biaslab/ReactiveMP.jl/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
@abpolym